pkg/data/data.go

```diff
@@ -199,6 +207,8 @@ type JobOffer struct {
 	// the user inputs to the module
 	// these values will power the go template
 	Inputs map[string]string `json:"inputs"`
+	// Expected input files
+	InputFiles InputFiles `json:"inputFiles"`
```

Comment:
Have you tested this with Anura? Theoretically this should not break Anura because it's making its' own module and then passing it to the solver; however, doesn't hurt to double check running Anura against this version of the solver locally just to make sure nothing breaks